### DISCUSSION TITLE
Problem with asi224

### DISCUSSION THREAD

**Discussion by dilsg** [aaronwmorris/indi-allsky#1071] [General]
I installed the system with indiserver. The system detects my asi224 cam. It is connected with my raspberry 4B on the usb-c port.
But I get no images. In teh log-file there is:
#281: Capture worker exception: RuntimeError: Error while changing property CCD_CONTROLS

Has anyone an idea?

**Comment by aaronwmorris** [comment]
I have been working with someone else for the past 3 days on this issue.  There is a bug with INDI 2.0.5 and some ZWO cameras, like the ASI224.

https://github.com/aaronwmorris/indi-allsky/wiki/INDI-2.0.5-notes

The fix is a setting for the camera.

```json
{
    "PROPERTIES" : {
        "CCD_CONTROLS" : {
            "GPSStartLine" : 0
        }
    },
    "SWITCHES" : {}
}
```

**Reply by aaronwmorris** [reply]
I have opened the following issue with the INDI development team regarding this issue.

https://github.com/indilib/indi-3rdparty/issues/867

**Comment by dilsg** [comment]
Hi - and juhu. It works and I get images.
Thank you.
